helm-chart/renku-notebooks/requirements.yaml

```diff
@@ -1,4 +1,8 @@
 dependencies:
 - name: amalthea
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: 0.2.2
+  version: "0.1.3"
+- name: dlf-chart
+  repository: "https://swissdatasciencecenter.github.io/datashim/"
+  version: "0.1.1-rc7"
```

Comment:
Feel free to publish an non-rc version of this chart before we merge this.

Reply:
I do not want to publish something like `0.1.1` because datashim.io do not publish their helm charts. I forked their repo in our org and added actions to publish the helm charts. So I am keeping the regular full release semvers to match regular releases from datashim.io. So I can publish something like `0.1.1-renku-1` or something along those lines. As long as it is a valid semver that helm likes. Helm can get quite picky about this.

Reply:
Do you expect datashim to publish a helm chart themselves anytime soon? Otherwise I I think publishing a datashim chart on the renku helm repo with say version `0.0.1` or `0.1.0` in which you refer to the latest datashim release as appVersion is acceptable. I'd expect chart and app version to diverge anyway, especially if we maintain the chart while they maintain the application.

Reply:
>Do you expect datashim to take your changes upstream and publish a helm chart themselves anytime soon?

No this will not happen anytime soon. They do not publish their helm charts at all and are fully missing any CI pipeline to even publish their images.

Also the PRs that I have submitted to datashim change both the helm chart and the app. So I will figure out what is the best way to name the chart.

Reply:
Honestly I think the easiest thing to do is to just release a chart with tag with `rc` or something like what I just committed. Let me know what you think. The datashim helm charts are not the simplest. It is actually one parent helm chart with several children and some interesting templating. So I am not sure it is worth trying to figure all that out. I have already spent some time just to get the CI workflows that publish the helm charts at our fork over at https://github.com/SwissDataScienceCenter/datashim

Reply:
Ok this was mostly due to a misunderstanding between me and Andreas on what datashim provides. The provide a whole helm chart but do not publish it or the images. We synced over slack and we are on the same page now.